docs/jwst/references_general/apcorr_reffile.inc

```diff
@@ -278,7 +278,7 @@ corrections for different radii.
 |            |       | apcorr_err  | float     | 2D array   | unitless |
 +------------+-------+-------------+-----------+------------+----------+
 
-:Data model: `~jwst.datamodels.NRSIFUApcorrModel`
+:Data model: `jwst.datamodels.NRSIFUApcorrModel <https://stdatamodels.readthedocs.io/en/latest/api/jwst.datamodels.NrsIfuApcorrModel.html>`_
```

Comment:
This link s pointing to a 404 for me. I think the right link is https://stdatamodels.readthedocs.io/en/latest/api/stdatamodels.jwst.datamodels.NrsIfuApcorrModel.html